src/utils/setup/fetch-build-time-media.js

```diff
@@ -3,10 +3,11 @@ import { STITCH_AUTH_APP_ID } from '../../constants';
 import { parsePodcasts } from '../parse-podcasts';
 import { transformTwitchResponse } from '../transform-twitch-response';
 import { transformYoutubeResponse } from '../transform-youtube-response';
+import memoizerific from 'memoizerific';
 
 const MAX_RESULTS = 5;
 
-export const fetchBuildTimeMedia = async () => {
+const fetchMedia = async () => {
```

Comment:
Ah line 13 is where I believe we get only a few of the videos

Reply:
Nice catch! This is reason why we had not all pages!